compiler/aarch64/codegen/BinaryEvaluator.cpp

```diff
@@ -1503,13 +1504,24 @@ generateUBFMForMaskAndShift(TR::Node *shiftNode, TR::CodeGenerator *cg)
 
          if (isRightShift)
             {
-            if (shiftNode->getOpCode().isShiftLogical())
+            TR::Register *shiftSrcReg = sreg;
+               if (shiftNode->getOpCode().isShiftLogical())
```

Comment:
Please correct the indentation of this line.

Reply:
Fixed it.